buddyboss/buddyboss-platform
# PROD-775 Group & Forum settings field will not show in activity settings when group & forum deactivated

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2725  .

### How to test the changes in this Pull Request:

1. Go to BuddyBoss setting components and disabled Social Groups and Forum Discussions modules.
2. Go to BuddyBoss Activity settings tab.
3. Find the Posts in Activity Feeds settings section.
4. Now you will not see, some of the setting options like Member creates a group, Member joins a group, Group details are updated, Member creates a forum discussion, Member replies to a forum discussion that are dependent on Group and Forum modules.

### Proof Screenshots or Video

https://www.loom.com/share/f7507297a620450f8e485b0b1c25cd87

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
